src/expression/index.ts

```diff
@@ -41,7 +41,7 @@ export type Feature = {
             'max': string;
         };
     };
-    readonly geometry?: Array<Array<Point>>;
+    readonly geometry?: Array<Array<Point2D>>;
```

Comment:
I'm not sure what problems this week cause when imported in the main maplibre-gl repo.
Is there a way to use geojson feature here and extend it? The definition of geometry here is incorrect in general I think as fir example for point geometry you only have one a single point, don't you?

Reply:
I hoped we could use a regular geojson instead of array of array of x,y, but that probably won't work...

Reply:
@HarelM Why not?